music-library/src/controllers/album.js

```diff
@@ -0,0 +1,91 @@
+const db = require('../db/index');
```

Comment:
This should be reformatted to use 2 spaces. You can do this by going into your IDE settings and searching for 'tab width', setting this to 2, and while you're there you can also search for 'format on save' and checking that box too, which will format the file you're working on when you edit it so you don't even have to remember to do it!